jormungandr/src/explorer/graphql/scalars.rs

```diff
@@ -69,6 +96,12 @@ impl TryFrom<EpochNumber> for blockcfg::Epoch {
     }
 }
 
+impl From<u64> for BlockCount {
+    fn from(number: u64) -> BlockCount {
+        BlockCount(format!("{}", number))
```

Comment:
I wonder if juniper requires these newtypes to wrap strings, or these can be the wrapped integers represented with `.to_string()` in the GraphQL-facing APIs as necessary.

Reply:
I can do that, it just requires a bit more of boilerplate, as I can't use the derive macro. I can wrap whatever type I want, I just need to provide the serialization and deserialization functions to the `juniper::graphql_scalar!` macro. (Or maybe there is a custom attribute for the derive, but I didn't find it in the documentation)